Dockerfile.test

```diff
@@ -0,0 +1,24 @@
+# Use the official PostgreSQL 16.1 image as the base image
+ARG PG_VERSION=16.1
+FROM postgres:${PG_VERSION}-alpine
+
+ARG PGTAP_VERSION=v1.3.2
+ENV DOCKERIZE_VERSION=v0.6.1
```

Comment:
@blazejpass I think you don't need it anymore as you've got rid of the `dockrize` right?